entries/p/pixiv.net.json

```
@@ -5,6 +5,10 @@
       "twitter": "pixiv_en",
       "email": "support@pixiv.net"
     },
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://www.pixiv.help/hc/en-us/articles/20876082054553-What-is-two-factor-authentication-",
```

Just like @fletcherdodd commented (although I can't seem to find that comment now?) the part to the right of the numbers isn't needed and can be removed.
```suggestion
    "documentation": "https://www.pixiv.help/hc/en-us/articles/20876082054553",
```